src/wallet/rpcwallet.cpp

```diff
@@ -3463,57 +3476,112 @@ UniValue z_listreceivedbyaddress(const UniValue& params, bool fHelp)
     }
 
     UniValue result(UniValue::VARR);
-    std::vector<SproutNoteEntry> sproutEntries;
-    std::vector<SaplingNoteEntry> saplingEntries;
-    pwalletMain->GetFilteredNotes(sproutEntries, saplingEntries, fromaddress, nMinDepth, false, false);
 
     std::set<std::pair<PaymentAddress, uint256>> nullifierSet;
     auto hasSpendingKey = boost::apply_visitor(HaveSpendingKeyForPaymentAddress(pwalletMain), zaddr);
     if (hasSpendingKey) {
         nullifierSet = pwalletMain->GetNullifiersForAddresses({zaddr});
     }
 
+    NotesFilters filters;
```

Comment:
```suggestion
    NotesFilter filter;
```
It's just one filter (maybe on multiple conditions).

Reply:
https://github.com/zcash/zcash/pull/4399/commits/53a8bd4fa62e06e4fd26d545f8cf4241094fbd47